command/service_delete_test.go

```diff
@@ -64,6 +64,7 @@ func TestServiceDeleteCommand_Run(t *testing.T) {
 	require.NoError(t, err)
 	registerCode := waitForSuccess(ui, client, fullId, t, regResp.EvalID)
 	require.Equal(t, 0, registerCode)
+	require.Equal(t, 0, waitForSuccess(ui, client, fullId, t, regResp.EvalID))
```

Comment:
Isn't this exactly the same as the two lines above it?

Reply:
Ohhh yeah! Sorry about that, tried to rush.